### PR TITLE
Change UTXO response as Stoa changes

### PR DESCRIPTION
**Pull request by MichaelKim20**
I added fields (block height, block time, unlock time) to the response on an endpoint /utxo:address

Relates to https://github.com/bpfkorea/stoa/pull/240

### CODE REVIEW

File: src/modules/net/response/UnspentTxOutput.ts [hunk 1]
```diff
@@ -40,14 +40,33 @@ export class UnspentTxOutput
      */
     amount: bigint;
 
+    /**
+     * Time at which the output of the transaction becomes available
+     */
+    unlock_time: number;
```
Comment (TrustHenry) [inline src/modules/net/response/UnspentTxOutput.ts:46]: This definition is ambiguous.

Comment (MichaelKim20) [reply src/modules/net/response/UnspentTxOutput.ts:46]: What makes you think so?

Comment (TrustHenry) [reply src/modules/net/response/UnspentTxOutput.ts:46]: I don't think we need to make this vague definition because there is unlock_height.

Comment (MichaelKim20) [reply src/modules/net/response/UnspentTxOutput.ts:46]: I'd like to remove it.

Comment (MichaelKim20) [reply src/modules/net/response/UnspentTxOutput.ts]: I changed it

**Review by TrustHenry — APPROVED** [review]

**Pull request merged**